scripts/base_network.py

```diff
@@ -548,6 +555,8 @@ def base_network():
 
     _apply_parameter_corrections(n)
 
+    _apply_sardinia_corsica_link_corrections(n)
+
```

Comment:
```suggestion
```
Now, `n = _remove_unconnected_components(n)` has to be placed below `_apply_parameter_corrections(n)` and bus '7276' is removed